src/detectors/__MOCKS__/Golang/BackendGoDetector.mock.ts

```diff
@@ -0,0 +1,24 @@
+import { IProjectComponentDetector } from '../../IProjectComponentDetector';
+import {
+  LanguageAtPath,
+  ProjectComponent,
+  ProjectComponentFramework,
+  ProjectComponentPlatform,
+  ProjectComponentType,
+} from '../../../model';
+import { injectable } from 'inversify';
+
+@injectable()
+export class BackendJavaDetector implements IProjectComponentDetector {
+  async detectComponent(langAtPath: LanguageAtPath): Promise<ProjectComponent[]> {
+    return [
+      {
+        path: langAtPath.path,
+        language: langAtPath.language,
+        framework: ProjectComponentFramework.UNKNOWN,
+        platform: ProjectComponentPlatform.BackEnd,
+        type: ProjectComponentType.Application,
+      },
+    ];
+  }
+}
```

Comment:
What is this mock for? Are you using it anywhere? I don't see it and I don't see the purpose.

Reply:
I referred to existing mocks at `/src/detectors/__MOCKS__` and I saw both `/Java/BackendJavaDetector.mock.ts` `/JavaScript/BackendJavaScriptDetector.mock.ts` so I followed.

After investigating, I believe both are not used. CMIIW but I think the idea is to group language usage based on the common framework used e.g in this case, usage of `express` indicates it's a backend JS app? https://github.com/DXHeroes/dx-scanner/blob/da6b55e706318037d275d423662383cef01401fa/src/detectors/__MOCKS__/JavaScript/BackendJavaScriptDetector.mock.ts#L18 

I can remove this if it's not necessary

Reply:
Oh, I see. I think it's just a forgotten code. Maybe we used to use that for some tests. You can remove it. You can also remove the unused BackendJavaScriptDetector and BackendJavaDetector. That would be awesome. :)

Reply:
deleted those